tests/Agent/IntegrationTests/IntegrationTestHelpers/AgentLogFile.cs

```diff
@@ -49,10 +49,12 @@ public AgentLogFile(string logDirectoryPath, ITestOutputHelper testLogger, strin
                 }
 
                 Thread.Sleep(TimeSpan.FromSeconds(1));
-            } while (timeTaken.Elapsed < timeout);
+            } while (timeTaken.Elapsed < timeout && logFileExpected);
```

Comment:
I realize the scenario we're fixing is unlikely to fail, but if we were to create a log file when we weren't expecting to, I'm not sure 1 second would be long enough to find it. Maybe we could just pass in a shorter timeout?

Reply:
Well, the logic in AgentLogFile.cs doesn't throw an exception if it finds a log when it wasn't expecting to.  Some of the tests in this namespace have assertions that specifically make sure that log files aren't created when/where they shouldn't be based on config.